package.json

```diff
@@ -24,18 +25,19 @@
     "lodash.merge": "^4.6.2",
     "numeral": "^2.0.6",
     "react": "^18.2.0",
+    "react-app-rewired": "^2.2.1",
     "react-dom": "^18.2.0",
     "react-router-dom": "6.4.2",
     "react-scripts": "5.0.1",
     "recharts": "2.1.15",
     "wagmi": "^0.12.1"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-app-rewired start",
     "hardhat": "npx hardhat node",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
+    "build": "react-app-rewired build",
+    "test": "react-app-rewired test  --transformIgnorePatterns 'node_modules/(?!@wagmi)/'",
+    "eject": "react-app-rewired eject",
```

Comment:
If this is changed for the whole project, we should watch for side effects on regular vercel hosting. On the other hand we hopefully move away from create-react-app soon anyway.

Reply:
Yes this replaces the `react-scripts` / create-react-app with `react-app-rewired` for the whole project.

We should maybe do some extra testing that none of the existing features break after this update.

Reply:
But yes I agree that this is only a temporary fix until we switch to next.js